### DISCUSSION TITLE
[FR] Adding Graph Cards

### DISCUSSION THREAD

**Discussion by virtualvoyager367** [Clooos/Bubble-Card#991] [Feature requests]
What if there were graphs like in [Mini Graph Card](https://github.com/kalkih/mini-graph-card) but styled like a Bubble Card.

**Comment by ChipSkylark37** [comment]
Would love this! I'd use it to help track the temperature change of HVAC / Sensors throughout the day, or electrical usage spikes when those same systems are running.

**Comment by Belt3r84** [comment]
Yes, I would also love this!

**Comment by Nemesis24** [comment]
+1

**Comment by Menelao147** [comment]
+1

**Comment by baerakafubu** [comment]
Would love this too but consider that the performance could drop. Mini graphic card or mini media player card are efficient with resources which is great

**Comment by kollock** [comment]
For sure, would love a graph card.

**Comment by korniza** [comment]
+1

**Comment by lpt2007** [comment]
+1

**Comment by Clooos** [comment]
Hi! This still needs a lot of work but here's a preview 😄

![Screenshot_2025-12-10-09-39-40-715_io homeassistant companion android-edit](https://github.com/user-attachments/assets/9ba22303-f16d-4471-a4d4-189c3bc5db71)

**Reply by GohanEhv** [reply]
Would love a graph card! Maybe you can get some inspiration from an old dashboard I used. (It's all Bubble now 😉). Combination of custom-stack-in-card, mushroom card, mini-graph card, and card-mod. I wanted the graphs to show their color based on their changing state. Mini-graph can't do that with two entities, only one. But with some help, I got it working. 

(Ignore the faults in it, didn't update or correct faults for a long time now)
<img width="496" height="308" alt="overview temp hum" src="https://github.com/user-attachments/assets/63d94968-aee7-46e3-83a8-16e3f6519bc6" />

**Reply by oeiber** [reply]
> Hi! This still needs a lot of work but here's a preview 😄
> 
> 
> 
> ![Screenshot_2025-12-10-09-39-40-715_io homeassistant companion android-edit](https://github.com/user-attachments/assets/9ba22303-f16d-4471-a4d4-189c3bc5db71)
> 
> 
> 
> 

Hey @Clooos!
Great work! It looks fantastic!
Currently I'm also using energy statistic cards, because they're responding to energy_date_selection. This gives me the ability to show values in a specific time range, for example mean of outside temperature or charged kwh for my ev. Would it be possible to implement this, as well?
Thank you!

**Comment by kollock** [comment]
Let's go!
